Resources/Prototypes/Entities/Clothing/OuterClothing/specific.yml

```diff
@@ -18,3 +18,35 @@
       interfaces:
         enum.ChameleonUiKey.Key:
           type: ChameleonBoundUserInterface
+
+#Chameleon web vest
+- type: entity
+  parent: ClothingOuterStorageBase
+  id: WebVestChameleon
+  name: vest
+  description: A thick vest with a rubbery, water-resistant shell.
+  suffix: Armored, Chameleon
+  components:
+    - type: Tag
+      tags: [] # ignore "WhitelistChameleon" tag
+    - type: Sprite
+      sprite: Clothing/OuterClothing/Vests/vest.rsi
+    - type: Clothing
+      sprite: Clothing/OuterClothing/Vests/vest.rsi
+    - type: ChameleonClothing
+      slot: [outerClothing]
+      default: ClothingOuterVest
+    - type: UserInterface
+      interfaces:
+        enum.ChameleonUiKey.Key:
+          type: ChameleonBoundUserInterface
+    - type: Armor
+      modifiers:
+        coefficients:
+          Blunt: 0.6 #ballistic plates = better protection
+          Slash: 0.6
+          Piercing: 0.3
+          Heat: 0.9
```

Comment:
This is identical to a web vest, making it a direct upgrade (apart from cost).
There's not really any sacrifice you have to make, especially if you're like me and barely use TC anyway.

Reply:
My reasoning was to give protection to stealth syndies, Security can still tell that something is wrong if a target seems too tanky. There is no reason for a syndicate that's going loud to use the chameleon version. To put off Sec's sens until they get a few shots off they can easily have a webbing or chest rig (Several mapped in gamer loot locations, or just take from a dead seccie)